src/VGriveClient.vala

```diff
@@ -191,21 +191,22 @@ namespace App {
             // Check if we have changes in files and sync them
             if (this.is_syncing ()) {
                 this.log_level=0;
-                this.check_deleted_files ();
                 this.check_remote_files (this.main_path);
                 this.check_local_files (this.main_path);
+                this.check_deleted_files ();
                 this.log_level=1;
-                if (this.is_syncing ()) {
```

Comment:
I put the `if` to detect earlier if the user has pushed the button "Stop syncing". Without this `if` the user will hve to wait all changes to be checked before stoping the sync. I think we should mantain the `if`.

Reply:
I understand. I'll put back then